package.json

```diff
@@ -97,5 +98,10 @@
       "js",
       "jsx"
     ]
+  },
+  "targets": {
```

Comment:
This is to inform Parcel that it can ignore the package.json since we're building a demo app